troubleshooting/index.md

```diff
@@ -42,7 +42,7 @@ Check the log file for errors
     see any error reports.
 
 Print all the things
-:   Adding [`print` statements](/docs/python/tutorial/inputoutput.html)
+:   Adding `print` statements
```

Comment:
They aren't statements

Reply:
Form the commit message:

> I've left in place the terminology of "print statements" because:
> a) it's teechnically still correct -- `print(x)` _is_ a statement
> b) this is easy to change later (at no additional effort) if we decide we really want to
> c) I think "print statement" sounds nicer than "print funtion call"

While `print` isn't a statement `print(x)` is.

Do you have a suggestion for a better terminology here?